setup.py

```diff
@@ -44,6 +44,7 @@
         "numpy>=1.19",
         "pandas>1.0",
         "scipy>=1.7",
+        "snakemake>=6.0",
```

Comment:
We don't use Conda/environment.yml to manage dependencies for MicroHapulator, so required Python packages need to be added here.

Reply:
It seems FastQC needs to be included as well. I created a fresh environment for testing and went through installation process. The Snakelike fails when trying to do FastQC.

Reply:
Unlike Snakemake, FastQC is not a Python package and can't be installed from PyPI with pip (which is what `setup.py` configures). For users, the only supported installation method is Conda, so when we push the new release I'll have to update the config for the MicroHapulator Bioconda package. Even so, we might want to update the documentation to list (at least) the non-Python dependencies.

Reply:
Fixed in a1c236c12be95eaa84d89b66eef6ff469d9abff1.